frontend/src/api/files.js

```diff
@@ -77,8 +77,13 @@ export function download (format, ...files) {
   if (format !== null) {
     url += `algo=${format}&`
   }
+  if (store.state.jwt !== ''){
+    url += `auth=${store.state.jwt}`
```

Comment:
If both jwt and token are not empty, the generated URL will be incorrect
```suggestion
    url += `auth=${store.state.jwt}&`
```